src/main/java/core/basesyntax/service/impl/ReportServiceCsvImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.dao.FruitStorageDao;
+import core.basesyntax.service.ReportService;
+import java.util.Map;
+
+public class ReportServiceCsvImpl implements ReportService {
```

Comment:
The class name 'ReportServiceCsvImpl' is descriptive, but ensure that it reflects the fact that this implementation is specific to generating reports for the CSV format. If the report format may change or if multiple formats are needed in the future, consider renaming or refactoring to accommodate that.